cosmwasm/contracts/outpost/src/execute.rs

```diff
@@ -0,0 +1,83 @@
+use crate::{
+    msg::{Callback, ExecuteMsg, Wasm, WasmHookExecute},
+    state::CONFIG,
+    ContractError,
+};
+use cosmwasm_std::{Addr, Coin, DepsMut, Response, Timestamp};
+
+// IBC timeout
+pub const PACKET_LIFETIME: u64 = 604_800u64; // One week in seconds
+
+//#[cfg(feature = "callbacks")]
```

Comment:
Ah interesting, hadn't thought of this potentially needing to be feature flagged